grab-objects/src/index.ts

```diff
@@ -0,0 +1,104 @@
+import {
+  engine,
+  GltfContainer,
+  Entity,
+  InputAction,
+  inputSystem,
+  PointerEventType,
+  PointerHoverFeedback,
+  Transform,
+  AudioSource,
+  CameraModeArea,
+  CameraType
+} from '@dcl/sdk/ecs'
+import { Quaternion, Vector3 } from '@dcl/sdk/math'
+
+export * from '@dcl/sdk'
+
+// Force camera first person
+CameraModeArea.create(engine.RootEntity, {
+  area: Vector3.create(32, 32, 32),
+  mode: CameraType.CT_FIRST_PERSON
+})
+
+// Base
+const base = engine.addEntity()
+GltfContainer.create(base, { src: `models/baseLight.glb` })
+
+// Configuration
+const Z_OFFSET = 1.5
+const GROUND_HEIGHT = 0.55
+const state = {
+  grabbed: false
+}
+const toggleGrabbed = () => {
+  state.grabbed = !state.grabbed
+  const sound = state.grabbed ? `sounds/put-down.mp3` : `sounds/pick-up.mp3`
+  playSound(sound)
+
+  return state.grabbed
+}
+
+// Sounds
+export function playSound(audio: string, loop: boolean = false) {
+  const entity = engine.addEntity()
```

Comment:
we are creating a new entity every time we click the box.
Maybe we can add a query to retrieve the AudioSource component and update the value instead of creating a new entity

Reply:
fixed 👍